libnullpay/src/utils/logger.rs

```diff
@@ -1,3 +1,6 @@
+#![allow(dead_code)]
```

Comment:
Please add reference to ticket in Jira devoted to removing this hotfix.
If possible and only few functions are unused, please move global directive to particular functions.

Reply:
After new changes below it's seems like these 2 allows are not required now, isn't it?

Reply:
agreed.